### PR TITLE
PLAT-132839: Show back button on Input

### PR DESCRIPTION
**Pull request by YunhoLee0**
### Checklist

* [x] I have read and understand the [contribution guide](http://enactjs.com/docs/developer-guide/contributing/)
* [x] A [CHANGELOG entry](http://enactjs.com/docs/developer-guide/contributing/changelogs/) is included
* [x] At least one test case is included for this feature or bug fix
* [x] Documentation was added or is not needed
* [ ] This is an API breaking change

### Issue Resolved / Feature Added
Need to add a back button in the Input

### Resolution
Migrate back button on Input patches from 2021 NMRM TV ("release/1.4.3-touch.x" branch)

### Links
PLAT-132839

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -6,6 +6,7 @@ The following is a curated list of changes in the Enact sandstone module, newest
 
 ### Added
 
+- `sandstone/Input` to add a back button and props `backButtonAriaLabel`, `noBackButton` to customize a back button aria label and to not show a back button
```
Comment (MikyungKim) [inline CHANGELOG.md:9]:
```suggestion
- `sandstone/Input` a back button and props `backButtonAriaLabel` and `noBackButton`
```

**Review by MikyungKim — COMMENTED** [review]
Please add screenshot tests for `noBackButton` prop.

File: CHANGELOG.md (hunk 1 above)
Comment (YunhoLee0) [reply CHANGELOG.md]: :OK:

**Review by MikyungKim — APPROVED** [review]
LGTM

**Pull request merged**